hail/python/test/hail/methods/test_qc.py

```diff
@@ -259,3 +259,13 @@ def test_sample_and_variant_qc_call_rate(self):
 
         assert mt.aggregate_cols(hl.agg.all(hl.approx_equal(mt.sample_qc.call_rate, mt.sample_qc.n_called / n_rows)))
         assert mt.aggregate_rows(hl.agg.all(hl.approx_equal(mt.variant_qc.call_rate, mt.variant_qc.n_called / n_cols)))
+
+    def test_summarize_variants_ti_tv(self):
+        mt = hl.import_vcf(resource('sample.vcf'))
+        # check that summarize can run with the print control flow
+        hl.summarize_variants(mt, handler=lambda s: ())
+        r = hl.summarize_variants(mt, show=False)
+        assert r['allele_types'] == {'Deletion': 27, 'Insertion': 18, 'SNP': 301}
+        assert r['contigs'] == {'20': 346}
+        assert r['n_variants'] == 346
+        assert r['r_ti_tv'] == 2.5
```

Comment:
Should we test `allele_counts` field here too?

Reply:
ah, sure